functions/content.js

```diff
@@ -101,7 +101,10 @@ module.exports = async function content({ page, context }) {
   }
 
   if (waitFor) {
-    if (typeof waitFor === 'string') {
+    if (typeof waitFor === 'object') {
```

Comment:
`Joi` validates that this is indeed an object prior to this execution, so we don't need to worry about Arrays or `null` here since they show up as Objects during typeof checks.